rules/catch-error-name.js

```diff
@@ -41,7 +41,7 @@ const create = context => {
 	};
 	const {name: expectedName} = options;
 	const ignore = options.ignore.map(
-		pattern => pattern instanceof RegExp ? pattern : new RegExp(pattern, 'u'),
+		pattern => Object.prototype.toString.call(pattern) === '[object RegExp]' ? pattern : new RegExp(pattern, 'u'),
```

Comment:
I don't think this is improvement here.

Reply:
So, should we ignore the rule in this line?